spec/features/scenarios/changes_of_circumstance/onboarding_a_deferred_participant_spec.rb

```diff
@@ -92,13 +107,25 @@ def when_context(scenario)
         context "Then the Original SIT" do
           subject(:original_sit) { "Original SIT" }
 
-          it { should_not find_participant_details_in_school_induction_portal "the Participant" }
+          it do
+            should_not find_participant_details_in_school_induction_portal "the Participant",
```

Comment:
don't bother changing this now but for future reference my understanding is the `should` syntax is the old way and will be removed at some point in the future. https://rspec.rubystyle.guide/#use-expect

Reply:
changed to expected_to